### PR TITLE
feat(NOJIRA-123): Add `onStarted` callback for "submission start" event

### PR DESCRIPTION
**Pull request by mathio**
Preview: https://worried-florentine-ambert.glitch.me/

### CODE REVIEW

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `e1db4945dda62346b1777b736b0de3143cd3b945` [here](https://embed.typeform.com/e1db4945dda62346b1777b736b0de3143cd3b945/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `890e169a5ae6792efc54a58854a6d29ab956a5c7` [here](https://embed.typeform.com/890e169a5ae6792efc54a58854a6d29ab956a5c7/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `23758d9b0fe89c5a7c41bf757a8d68de51453e4a` [here](https://embed.typeform.com/23758d9b0fe89c5a7c41bf757a8d68de51453e4a/embed.js).

File: docs/callbacks.md [hunk 1]
```diff
@@ -50,6 +51,36 @@ Or in HTML:
...
+```
+
+Or in HTML:
+
+```html
+<div data-tf-widget="<form-id>" data-tf-on-started="started"></div>
+<script src="//embed.typeform.com/next/embed.js"></script>
+<script>
+  // this function needs to be available on global scope (window)
+  function started({ formId }) {
```
Comment (Thr44) [inline docs/callbacks.md:78]: we need to add the responseId
```suggestion
  function started({ formId, responseId }) {
```

Comment (mathio) [reply docs/callbacks.md]: Thanks, I missed this lat night :)

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `9145da23f04bdf8bd2fa050b99171b4852995904` [here](https://embed.typeform.com/9145da23f04bdf8bd2fa050b99171b4852995904/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `47b49c586c90396e3e388fc0db4bd416775d7677` [here](https://embed.typeform.com/47b49c586c90396e3e388fc0db4bd416775d7677/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `3473553e7a0c2634d73356ceeecf3ce921359e88` [here](https://embed.typeform.com/3473553e7a0c2634d73356ceeecf3ce921359e88/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `3b96f2f9c44895d4521d315e33ab3937a649f497` [here](https://embed.typeform.com/3b96f2f9c44895d4521d315e33ab3937a649f497/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `302dbddf4ad40446357fd90c6a2ae2d8a9c7a0a2` [here](https://embed.typeform.com/302dbddf4ad40446357fd90c6a2ae2d8a9c7a0a2/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `a09ecd84bb93194201921ca956ee73efebd84552` [here](https://embed.typeform.com/a09ecd84bb93194201921ca956ee73efebd84552/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `48ddcfae88c3b29b6454d81a3fa0d62382eec815` [here](https://embed.typeform.com/48ddcfae88c3b29b6454d81a3fa0d62382eec815/embed.js).

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `b0f48dc05badf5bcadd79b02da3e1767aeaf9f60` [here](https://embed.typeform.com/b0f48dc05badf5bcadd79b02da3e1767aeaf9f60/embed.js).

**Review by mumpo — APPROVED** [review]

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `5efa2a786f19be5099869894df53d14c9e6fa6cb` [here](https://embed.typeform.com/5efa2a786f19be5099869894df53d14c9e6fa6cb/embed.js).

**Pull request merged**